libvmi/slat.h

```diff
@@ -0,0 +1,119 @@
+/* The LibVMI Library is an introspection library that simplifies access to
+* memory in a target virtual machine or in a file containing a dump of
+* a system's physical memory.  LibVMI is based on the XenAccess Library.
+*
+* Copyright 2011 Sandia Corporation. Under the terms of Contract
```

Comment:
I don't think you are under contract with Sandia, so remove this block from here.